src/constants/common.ts

```diff
@@ -153,3 +153,5 @@ export const MAINNET_URL = `https://${config.BASE_URL}`
 export const TESTNET_URL = `https://${ChainName.Testnet}.${config.BASE_URL}`
 
 export const TYPE_ID_CODE_HASH = '0x00000000000000000000000000000000000000000000000000545950455f4944'
+
+export const NETWORK = config.CHAIN_TYPE === 'testnet' ? 'pudge' : 'minara'
```

Comment:
The network type is defined at https://github.com/Magickbase/ckb-explorer-frontend/blob/develop/src/constants/common.ts#L21

Specific name `pudge` and `minara` could be defined in the node probe module(https://github.com/Magickbase/ckb-explorer-frontend/pull/279/files#diff-f4b4604e1382a061c19d32145849a51342ab599f4160fd30da496410a35ba673R1) because it's not a general value for other modules.